src/Symfony/Bundle/FrameworkBundle/Resources/config/form.xml

```diff
@@ -93,6 +93,7 @@
             <deprecated>The "%service_id%" service is deprecated since Symfony 3.1 and will be removed in 4.0.</deprecated>
         </service>
         <service id="form.type.file" class="Symfony\Component\Form\Extension\Core\Type\FileType" public="true">
+            <argument type="service" id="translator" on-invalid="ignore" />
             <deprecated>The "%service_id%" service is deprecated since Symfony 3.1 and will be removed in 4.0.</deprecated>
```

Comment:
Probably means we need to undeprecate the service, right?

Reply:
Yes!